docs/develop/reference/midnight-api/midnight-js/@midnight-ntwrk/midnight-js-fetch-zk-config-provider/classes/FetchZkConfigProvider.md

```diff
@@ -60,7 +60,7 @@ The endpoint to query for ZK artifacts.
 
 > **get**(`circuitId`): `Promise`\<[`ZKConfig`](../../midnight-js-types/interfaces/ZKConfig.md)\<`K`\>\>
 
-Retrieves all zero-knowledge artifacts produced by `compactc` for the given circuit.
+Retrieves all zero-knowledge artifacts produced by `compact` for the given circuit.
```

Comment:
These docs are copied into the docs tree from a different repository, there is no point updating them.

To me, I think it's irrelevant to this API how the ZK artifacts were produced (which tool produced them) --- it just doesn't matter and if we change it (like we did) we don't want to have to update these.

So they need a bigger rewrite than just this, and in a different repository.